src/Servers/Kestrel/test/InMemory.FunctionalTests/HttpsConnectionMiddlewareTests.cs

```diff
@@ -160,6 +165,37 @@ void ConfigureListenOptions(ListenOptions listenOptions)
             }
         }
 
+        [Fact(Skip = "https://github.com/dotnet/runtime/issues/40402")]
```

Comment:
You could mark this as Quarantined and then we'll unquarantine it once it starts passing.